package.json

```diff
@@ -180,470 +180,78 @@
     ]
   },
   "typings": "./handsontable.d.ts",
+  "handsontable": {
+    "copy": [
+      "dist/handsontable.css",
+      "dist/handsontable.full.css",
+      "dist/handsontable.full.js",
+      "dist/handsontable.full.min.css",
+      "dist/handsontable.full.min.js",
+      "dist/handsontable.js",
+      "dist/handsontable.min.css",
+      "dist/handsontable.min.js",
+      "dist/languages",
+      "dist/README.md",
+      "languages",
+      "base.d.ts",
+      "CHANGELOG.md",
+      "handsontable-general-terms.pdf",
+      "handsontable-non-commercial-license.pdf",
+      "handsontable.d.ts",
+      "LICENSE.txt",
+      "README.md"
+    ],
+    "exports": [
+      "./cellTypes/**/+(index|registry)*",
+      "./languages/!(all.js)",
+      "./editors/**/+(index|registry)*",
+      "./plugins/**/+(index|registry)*",
+      "./renderers/**/+(index|registry)*",
+      "./validators/**/+(index|registry)*",
+      "./i18n/+(index|registry)*",
+      "./i18n/languages/*",
+      "./dist/**/*.+(js|css)",
+      {
+        ".": {
+          "import": "./index.mjs",
+          "require": "./index.js"
+        }
+      },
+      {
+        "./base": {
+          "import": "./base.mjs",
+          "require": "./base.js"
+        }
+      },
+      {
+        "./languages/all": {
+          "import": "./languages/index.mjs",
+          "require": "./languages/all.js"
+        }
+      }
+    ],
+    "fields": [
+      "name",
+      "description",
+      "homepage",
+      "repository",
+      "bugs",
+      "author",
+      "version",
+      "main",
+      "module",
+      "jsnext:main",
+      "jsdelivr",
+      "unpkg",
+      "keywords",
+      "dependencies",
```

Comment:
@swistach WDYT about leaving the `devDependencies` field in the generated `package.json` file? Without that, I genuinely feel that we can lose some kind of transparency. In the https://www.npmjs.com/package/handsontable page (and similar pages that aggregate the npm packages) the "dependencies" tab will not reflect the whole truth. 

CC @wojciechczerniak

Reply:
Agree

Reply:
OK! Added in bbf6099